### DISCUSSION TITLE
Soft Serial on SDA / SCL pads

### DISCUSSION THREAD

**Discussion by carlybaby69** [iNavFlight/inav#6767] [Ideas]
Take the Matek 411 WSE for example. A tiny board with limited uarts that gets fitted to small planes - Zohd 250g, Drift etc. A lot of the applications for this board would mean most users are not fitting compasses / oled screens etc. Therefore I bet a lot of the SDA / SCL pads are unused. 

Wouldn't this therefore make a good candidate as a full soft serial?

The cheap omnibus F4 V3 you can use SDA / SCL as either that or a UART and it is very useful. 

Is it possible to use SDA / SCL pads as a soft serial? If so why don't we make it more of a universal soft serial option as I said I bet they are very often not used as SDA / SCL.

**Comment by JulioCesarMatias** [comment]
hi, I don’t know if any developer will work on it, because there are already 2 Uart and 2 Soft Serial, 4 serial communication ports in total, it’s quite a big number. pull-request
(if you are a programmer). Here is the pin configuration file for your FC >> https://github.com/iNavFlight/inav/blob/master/src/main/target/MATEKF411SE/target.c

**Comment by JulioCesarMatias** [comment]
the only modification currently for your FC was to add a control for VTX switches, it is not yet available in any version of iNAV, it may be in the next version 2.7 >> https://github.com/iNavFlight/inav/pull/6166
